### PR TITLE
HIR-487 and HIR-494 Celery setup and many to many mapping for scheduled reports and charts

### PR DESCRIPTION
**Pull request by swapnil95**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -11,4 +12,4 @@ MySQL-python==1.2.5
 gunicorn==19.6.0
 whitenoise==3.3.0
 PyJWT==1.4.2
-croniter==0.3.16
\ No newline at end of file
+croniter===0.3.16
```
Comment (dakshgautam) [inline requirements.txt]: why added extra '=' ?

File: squealy/models.py [hunk 2]
```diff
@@ -115,16 +115,27 @@ class ScheduledReport(models.Model):
...
+    def __unicode__(self):
+        return self.subject
+
+
+class ScheduledReportChart(models.Model):
+    '''
+        Many to many mapping between charts and cheduled reports
+    '''
+    chart = models.ForeignKey(Chart)
+    report = models.ForeignKey(ScheduledReport)
```
Comment (dakshgautam) [inline squealy/models.py]: Why related_name field not added in these foreign keys ?

File: squealyproj/celeryapp.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
+
+import os
+
+from celery import Celery
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'example.settings')
+
+from django.conf import settings  # noqa
```
Comment (dakshgautam) [inline squealyproj/celeryapp.py]: Move this import ub the file

File: squealyproj/settings.py [hunk 4]
```diff
@@ -11,6 +11,7 @@
 """
 
 import os
+import djcelery
 from .utils import extract_dj_database_urls
```
Comment (dakshgautam) [inline squealyproj/settings.py]: Add extra space .

File: squealyproj/settings.py [hunk 5]
```diff
@@ -156,3 +159,12 @@
 EMAIL_PORT = os.environ.get('EMAIL_PORT')
 DEFAULT_FROM_EMAIL = EMAIL_HOST_USER
 
+# Celery settings
+
```
Comment (dakshgautam) [inline squealyproj/settings.py]: Remove this extra space .

File: squealyproj/celeryapp.py (hunk 3 above)
Comment (swapnil95) [inline squealyproj/celeryapp.py]: On line 8 the settings are being set for the celery CLI. Hence after setting the DJANGO_SETTINGS_MODULE variable, I am importing the project settings for celery 

This is how they have suggested to [initialize celery.](http://docs.celeryproject.org/en/3.1/django/first-steps-with-django.html)

**Pull request merged**